manifests/init.pp

```diff
@@ -129,7 +130,8 @@
   }
 
   concat { $config_file:
-    notify  => Service[$service_name],
+    validate_cmd => '/sbin/unbound-checkconf %',
```

Comment:
This needs to take into consideration other OSs.  On OpenBSD for example, this is '/usr/sbin/unbound-checkconf'.